src/types.ts

```diff
@@ -0,0 +1 @@
+export type ReviewComment = { body: string; path: string; line: number };
```

Comment:
The `line` property is of type `number`, which might not be sufficient for all use cases. Consider using a more specific type like `LineNumber = number & { __brand: 'LineNumber' }` to prevent accidental misuse with other number values.